From: jiyun_ha <jiyun_ha@tmax.co.kr>
Date: Mon, 24 Jan 2022 16:18:41 +0900
Subject: [feat] appindicator : disable tray icon which is using xembed
 protocol

 - disable showing app icon that is using xembed protocol

[IMS] 272186
---
 extensions/appindicator/extension.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/extensions/appindicator/extension.js b/extensions/appindicator/extension.js
index 5a73398..f8af164 100644
--- a/extensions/appindicator/extension.js
+++ b/extensions/appindicator/extension.js
@@ -57,7 +57,7 @@ function maybeEnableAfterNameAvailable() {
 function enable() {
     isEnabled = true;
     maybeEnableAfterNameAvailable();
-    TopIcons.createTray();
+//    TopIcons.createTray();
 }
 
 function disable() {
